syntax/glsl.vim

```diff
@@ -26,15 +26,15 @@ syn region  glslPreProc         start="^\s*#\s*\(error\|pragma\|extension\|versi
 syn keyword glslBoolean true false
 
 " Integer Numbers
-syn match glslDecimalInt display "\(0\|[1-9]\d*\)"
-syn match glslOctalInt   display "0\o\+"
-syn match glslHexInt     display "0[xX]\x\+"
+syn match glslDecimalInt display "\(0\|[1-9]\d*\)[uUfF]\?"
+syn match glslOctalInt   display "0\o\+[uUfF]\?"
+syn match glslHexInt     display "0[xX]\x\+[uUfF]\?"
```

Comment:
`f` and `F` suffixes are redundant (at least right now) but this doesn't mean they can't be used.

Also I would not use wiki as the main source of truth (even if it's wiki on Khronos site). It's better to check GLSL specification (links to PDFs can be found [here](https://en.wikipedia.org/wiki/OpenGL_Shading_Language#References)). Correctness of GLSL code can be checked using [reference compiler from Khronos](https://www.khronos.org/opengles/sdk/tools/Reference-Compiler/).